src/main/kotlin/com/sourcegraph/cody/edit/exception/EditExecutionException.kt

```diff
@@ -0,0 +1,8 @@
+package com.sourcegraph.cody.edit.exception
+
+import com.sourcegraph.cody.edit.fixupActions.FixupUndoableAction
+
+class EditExecutionException(action: FixupUndoableAction, cause: Throwable) :
+    RuntimeException(cause) {
+  override val message: String = "Edit application failed for $action"
```

Comment:
I don't like this error message. It may suggest that there is an application (a program) which has just failed, or that we have failed to edit said application.

I would put it like this: `"Failed to apply the edit for $action"`, or perhaps "Could not apply the edit to ..." etc.